src/Illuminate/Contracts/Encryption/Encrypter.php

```diff
@@ -21,4 +21,19 @@ public function encrypt($value, $serialize = true);
      * @return mixed
      */
     public function decrypt($payload, $unserialize = true);
+
+    /**
+     * Generate a new key for the chosen cipher.
+     *
+     * @param  string  $cipher
```

Comment:
string|null

Reply:
This has been updated on the contract, `EncryptionManager`, and `OpenSslEncrypter`